internal/app/tfsec/checks/utils.go

```diff
@@ -17,3 +19,38 @@ func isBooleanOrStringTrue(val *parser.Attribute) bool {
 		return false
 	}
 }
+
+// isOpenCidr returns true if given attribute is an open CIDR block
+func isOpenCidr(attr *parser.Attribute, provider scanner.RuleProvider) bool {
+	if attr.Value().IsNull() {
+		return false
+	}
+
+	var cidrList []cty.Value
+	if attr.Type() == cty.String {
+		cidrList = []cty.Value{attr.Value()}
+	} else {
+		cidrList = attr.Value().AsValueSlice()
+	}
+
+	for _, cidr := range cidrList {
+		if cidr.Type() != cty.String {
+			continue
+		}
+
+		cidrStr := cidr.AsString()
+
+		switch provider {
```

Comment:
Hey @particleflux  - apologies for the delay on reviewing this. 

I've just been looking at the documentation and it doesn't seem "*" is valid for AWS or GCP, so there is nothing to be gained with this switch by the look of it. We can get away with

```go
if strings.HasSuffix(cidrStr, "/0" || cidrStr == "*" {
    return true
}
```

and remove the switch altogether?

Reply:
Changed, avoids unnecessary complexity :+1: 
Also rebased upon current master to have a clean commit history